Expensify/App
# Handle VBA errors with red field outlines

### Details
This PR adds red borders for any validation errors stemming from text inputs in the VBA flow.

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/171620

### Tests/QA
1. Navigate to `<baseURL>/bank-account/company`.
2. Input invalid data for the following fields and try to submit the form. Verify that red outlines AND growl messages occur.
    - Empty password
    - Invalid address street
    - Invalid address zip code
    - Invalid website URL
    - Invalid company tax ID
    - Invalid industry code
    - Invalid incorporation date
3. Navigate to `<baseURL>/bank-account/requestor`.
4. Input invalid data for any of the fields and try to submit the form. Verify that red outlines AND growl messages occur.
    - Empty date of birth
    - Invalid address street
    - Invalid address zip code
    - Invalid last 4 digits of SSN

### Tested On


### Screenshots


#### Web
<img width="1792" alt="Screen Shot 2021-08-09 at 7 23 23 AM" src="https://user-images.githubusercontent.com/31285285/128747693-5a986f92-a09a-4734-be50-a9439bb82841.png">

#### Mobile Web
<img width="358" alt="Screen Shot 2021-08-09 at 7 23 40 AM" src="https://user-images.githubusercontent.com/31285285/128747702-33f7db27-7773-4965-9b5f-022e3fac3105.png">

#### Desktop
<img width="1071" alt="Screen Shot 2021-08-09 at 7 56 10 AM" src="https://user-images.githubusercontent.com/31285285/128751704-1dfe8791-4c5c-447f-8aac-e0aa9d8f2aff.png">

#### iOS
![Simulator Screen Shot - iPad Pro (12 9-inch) (4th generation) - 2021-08-09 at 07 42 55](https://user-images.githubusercontent.com/31285285/128749993-5fdb7dda-25a9-47cc-addb-8518118a4a98.png)

#### Android
![Screenshot_1628531581](https://user-images.githubusercontent.com/31285285/128751354-ede47684-00e3-4cfc-95e9-ac516bc6423d.png)
